docs/public/config.toml

```diff
@@ -0,0 +1,21 @@
+# The URL the site will be built for
+base_url = "https://openmsupply.github.io/application-manager-web-app/"
+
+# Whether to automatically compile all Sass files in the sass directory
+compile_sass = true
+
+# Whether to build a search index to be used later on by a JavaScript library
+build_search_index = true
+
+theme = "zola_easydocs_theme"
+
+[markdown]
+# Whether to do syntax highlighting
+# Theme can be customised by setting the `highlight_theme` variable to a theme supported by Zola
+highlight_code = true
+
+[extra]
+# Put all your custom variables here
+logo = "https://easydocs.codeandmedia.com/logo.svg"
+release = "https://api.github.com/repos/openmsupply/application-manager-web-app/releases/latest"
```

Comment:
Showing the version of the latest `application-manager-web-app` release (currently showing `undefined` as there are no releases).